### PR TITLE
Release 2 CICD changes.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* /bcgov/entity###

*Description of changes:*

1. Set main branch as default branch for CICD;
2. Add commit ref and build date label into docker image;
3. Enable CD command in make file;
4. Move environment config to 1password;
5. Update the sbc common component to version 2.4.14

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "launchdarkly-js-client-sdk": "^2.18.3",
     "lodash": "^4.17.20",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.4.8",
+    "sbc-common-components": "2.4.14",
```
Comment (severinbeauvais) [inline package.json:27]: Please update the NR app version on line 3 as well (so we can easily tell what's deployed).

**Review by severinbeauvais — APPROVED** [review]

File: devops/vaults.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "application": [
+            "base",
+            "web-base",
+            "name-requests-web"
+        ]
+    },
+    {
+        "vault": "name-request",
+        "application": [
+            "namerequest-ui"
```
[the remark this answers is not in the document]
Comment (pwei1018) [reply devops/vaults.json:13]: it is "namerequest-ui" I set the name in 1password.

File: package.json (hunk 1 above)
Comment (pwei1018) [reply package.json:27]: done

Comment (severinbeauvais) [reply package.json:27]: Thanks.
Please also check in the updated package-lock.json.

Comment (pwei1018) [reply package.json:27]: Yes

**Pull request merged**